addons/better-featured-project/userscript.js

```diff
@@ -0,0 +1,55 @@
+export default async function ({ addon, global, console, msg }) {
+  function createBetterProfilePage(featuredThumb, featuredLink, featuredHeading, featuredTitle) {
+    document.documentElement.style.setProperty("--featured-thumb", `url("${featuredThumb}")`);
+    document.documentElement.style.setProperty("--refresh-notice", `\"${msg("refresh-notice")}\"`);
```

Comment:
This text is not visible with the "website dark mode" addon enabled. Maybe you could open a `prompt()` that asks the user if they want to refresh to see the changes?

Reply:
I figured out how to make it so that it automatically refreshes, which I think would be easier and quicker for the user, but I've also noticed that most of the text isn't even visible with the "website dark mode" addon enabled and "Scratch 2.0 -> 3.0" disabled, which should probably be reported as a separate issue.
![image](https://user-images.githubusercontent.com/38388911/101060502-7e2c5f80-355d-11eb-937a-bd2bf2c4cefb.png)